oshc/main/static/main/css/app.css

```diff
@@ -144,6 +152,8 @@ footer {
 
 footer .container {
   width: 70%;
```

Comment:
I think there is still some problem in the Footer section. _[Screenshot]_
Suggestions change `footer .container`'s width from `70%` to `100%`.

![screenshot from 2017-06-23 12-06-49](https://user-images.githubusercontent.com/9383897/27469383-e37f46a4-580c-11e7-9f94-f09972242b35.png)